pkg/frontend/validate.go

```diff
@@ -78,26 +79,48 @@ func (f *frontend) validateOpenShiftUniqueKey(ctx context.Context, doc *api.Open
 // prevent mischief
 var rxKubernetesString = regexp.MustCompile(`(?i)^[-a-z0-9.]{0,255}$`)
 
-func validateAdminKubernetesObjectsNonCustomer(method, groupKind, namespace, name string) error {
+func validatePermittedClusterwideObjects(gvr *schema.GroupVersionResource) bool {
+	validApiGroups := []string{
+		"nodes",
+		"aro.openshift.io",
+		"config.openshift.io",
+		"console.openshift.io",
+		"machine.openshift.io",
+		"machineconfiguration.openshift.io",
+		"imageregistry.operator.openshift.io",
+		"certificatesigningrequest.certificates.k8s.io",
+		"apirequestcounts.apirequestcount.openshift.io",
+		"rbac.authorization.k8s.io",
+		"authorization.openshift.io",
+	}
+	for _, apiGroup := range validApiGroups {
+		if gvr.Group == apiGroup {
+			return true
+		}
+	}
+	return false
```

Comment:
if this was a `map[string]bool` that'd reduce the complexity a little bit because we'd be able to do `return isValid[gvr.Group]`

Reply:
I had to update this function to check for two different conditions.

Reply:
@bennerv @facchettos Is this change something that is blocking, or can it be done in a follow up?

Reply:
I'd rather have it directly in it tbh, but I'm not the boss of aro so :D

Reply:
@facchettos Why? Is there a functional difference?